mwdb/web/src/components/Settings/Views/AttributeCreate.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from "react";
+
+import api from "@mwdb-web/commons/api";
+import { useViewAlert } from "@mwdb-web/commons/ui";
+
+export default function AttributeCreate() {
+    const viewAlert = useViewAlert();
+    const [values, setValues] = useState({
+        metakey: "",
+        label: "",
+        description: "",
+        template: "",
+        hidden: false,
+    });
+
+    function handleInputChange(event) {
+        const target = event.target;
+        const value =
+            target.type === "checkbox" ? target.checked : target.value;
+        const name = target.name;
+
+        setValues((prevState) => ({
+            ...prevState,
+            [name]: value,
+        }));
+    }
+
+    async function createAttribute() {
+        try {
+            await api.addMetakeyDefinition(
+                values.metakey,
+                values.label,
+                values.description,
+                values.template,
+                values.hidden
+            );
+            viewAlert.redirectToAlert({
+                target: `/admin/attribute/${values.metakey}`,
+                success: "Attribute created successfully.",
+            });
+        } catch (error) {
+            viewAlert.setAlert({ error });
+        }
+    }
+
+    return (
+        <div className="container">
+            <h2>Create new attribute</h2>
+            <form
+                onSubmit={(ev) => {
+                    ev.preventDefault();
+                    createAttribute();
+                }}
+            >
+                <div className="form-group">
+                    <label>Key</label>
+                    <input
+                        type="text"
+                        name="metakey"
+                        value={values.metakey}
+                        onChange={handleInputChange}
+                        className="form-control"
+                        required
+                        pattern="[a-z0-9_-]{1,32}"
+                    />
+                    <div className="form-hint">
+                        Key must contain only lowercase letters and digits, max
+                        32 characters allowed.
+                    </div>
+                </div>
+                <div className="form-group">
+                    <label>Label</label>
+                    <input
+                        type="text"
+                        name="label"
+                        value={values.label}
+                        onChange={handleInputChange}
+                        className="form-control"
```

Comment:
I tking that Label should be required as well as Key because as user you don't see key but label.
```suggestion
                        className="form-control"
                        required
```

Reply:
Key is shown If label is not defined. Some attribute keys (e.g. "password") doesn't require additional, user-friendly label